examples/Forms/MultiLineTextInput.md

```diff
@@ -0,0 +1,159 @@
+Basic multi-line text box
+
+```jsx
+import { MultiLineTextInput } from 'mark-one';
+import { useState } from 'react';
+
+const [value, setValue] = useState('CS50 is an on-campus and online introductory course on computer science taught at Harvard University and Yale University.');
+
+<MultiLineTextInput
+  value={value}
+  id="course-notes"
+  label="Course notes"
+  onChange={(event) => {
+    setValue(event.target.value);
+  }}
+/>
+```
+
+Multi-line text box with an error
+```jsx
+import { MultiLineTextInput } from 'mark-one';
+import { useState } from 'react';
+
+const [value, setValue] = useState('');
+
+<MultiLineTextInput
+  value={value}
+  id="course-notes"
+  label="Course notes"
+  errorMessage="Please provide a value for this field"
+  onChange={(event) => {
+    setValue(event.target.value);
+  }}
+/>
+```
+
+Multi-line text box with a hidden error
+```jsx
+import { MultiLineTextInput } from 'mark-one';
+import { useState } from 'react';
+
+const [value, setValue] = useState('');
+
+<MultiLineTextInput
+  value={value}
+  id="course-notes"
+  label="Course notes"
+  errorMessage="Please provide a value for this field"
+  hideError
+  onChange={(event) => {
+    setValue(event.target.value);
+  }}
+/>
+```
+
+
+Multi-line text box with a hidden label
+```jsx
+import { MultiLineTextInput } from 'mark-one';
+import { useState } from 'react';
+
+const [value, setValue] = useState('');
+
+<MultiLineTextInput
+  value={value}
+  id="course-notes"
+  label="Course notes"
+  isLabelVisible={false}
+  onChange={(event) => {
+    setValue(event.target.value);
+  }}
+/>
+```
+
+Multi-line text box with a hidden label
+```jsx
+import { MultiLineTextInput } from 'mark-one';
+import { useState } from 'react';
+
+const [value, setValue] = useState('');
+
+
+<MultiLineTextInput
+  value={value}
+  id="course-notes"
+  label="Course notes"
+  isLabelVisible={false}
+  onChange={(event) => {
+    setValue(event.target.value);
+  }}
+/>
+```
+
+Required multi-line text box
+```jsx
+import { MultiLineTextInput } from 'mark-one';
+import { useState } from 'react';
+
+const [value, setValue] = useState('');
+
+
+<MultiLineTextInput
+  value={value}
+  id="course-notes"
+  label="Course notes"
+  isLabelVisible={false}
+  isRequired
+  onChange={(event) => {
+    setValue(event.target.value);
+  }}
+/>
+```
```

Comment:
For the `TextInput,` when isRequired is true, there is a red asterisk that appears next to the label. I think it may be good to update this example to show that by making `isLabelVisible` true instead of false.

Reply:
You're right. Actually I just removed that prop altogether from this example, since it defaults to `true` anyway in 630d720